app/controllers/users_controller.rb

```diff
@@ -1,2 +1,6 @@
 class UsersController < ApplicationController
+  def trip
```

Comment:
what exactly is this method doing? and where would it be used?

Reply:
added routes users/:id/trips and method attached to it is users#trip so if we go to users/:id/trips we can see the trips belongs to users. it's just custom method and route other than 7 cruds. we can rename it to dashboard or something else.